sysutils/edid-decode/Portfile

```diff
@@ -0,0 +1,17 @@
+PortSystem       1.0
+
+name             edid-decode
+version          git-e48fb38
+categories       sysutils
+platforms        darwin
```

Comment:
`platforms darwin` is implicit in MacPorts now, and can be completely dropped.

Reply:
Fixed (d73a890).